workflow/scores.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import division
+
+import itertools
+import numpy as np
+
+from iou import cc_iou as iou
+
+
+def score_craters_on_patch(y_true, y_pred):
+    """
+    Main score
+
+    Parameters
+    ----------
+    y_true : list of tuples (x, y, radius)
+        List of coordinates and radius of actual craters in a patch
+    y_pred : list of tuples (x, y, radius)
+        List of coordinates and radius of craters predicted in the patch
+
+    Returns
+    -------
+    float : score for a given path, the higher the better
+
+    """
+    y_true = np.atleast_2d(y_true).T
+    y_pred = np.atleast_2d(y_pred).T
+
+    ospa_score = ospa(y_true, y_pred)
+
+    score = 1 - ospa_score
+
+    return score
+
+
+def score_iou(y_true, y_pred):
+    pass
+
+
+def score_completeness(y_true, y_pred):
+    pass
+
+
+def ospa(x_arr, y_arr, p_norm=1, cut_off=1):
+    """
+    Optimal Subpattern Assignment (OSPA) metric for IoU score
+
+    This metric provides a coherent way to compute the miss-distance
+    between the detection and alignment of objects.
+
+    The lower the value the smaller the distance.
+
+    Parameters
+    ----------
+    x_arr, y_arr : ndarray of shape (3, x)
+        arrays to compare
+    p_norm : int, optional (default is 1)
+        distance norm
```

Comment:
Can you explain this a bit more ?

Reply:
I don't know if it is actually relevant here. Maybe I should just discard totally the `p_norm` since it would change the range of output values. What do you think ?

Reply:
I didn't read the paper, so I don't know what is the purpose/effect of a different value than 1. But if we are only going to use 1, it indeed sounds like a good idea to remove it, as it in that case unnecessarily complicates the code